OSDependent/Unix/omnisci_path.cpp

```diff
@@ -26,10 +26,30 @@
 #include <boost/filesystem/path.hpp>
 
 #include "Logger/Logger.h"
+#ifdef ENABLE_EMBEDDED_DATABASE
+#ifndef _GNU_SOURCE
+#define _GNU_SOURCE
+#endif
+#include <dlfcn.h>
+#include <link.h>
+#endif
 
 namespace omnisci {
 
 std::string get_root_abs_path() {
+#ifdef ENABLE_EMBEDDED_DATABASE
+  void* const handle = dlopen(DBEngine_LIBNAME, RTLD_LAZY);
+  CHECK(handle);
+  const struct link_map* link_map = 0;
+  const int ret = dlinfo(handle, RTLD_DI_LINKMAP, &link_map);
+  CHECK_EQ(ret, 0);
+  CHECK(link_map);
+  /* Despite the dlinfo man page claim that l_name is absolute path,
+     it is so only when the location path to the library is absolute,
+     say, as specified in LD_LIBRARY_PATH. */
+  boost::filesystem::path abs_exe_dir(boost::filesystem::absolute(
+      boost::filesystem::canonical(std::string(link_map->l_name))));
+#else
```

Comment:
So, no Windows support so far?

Reply:
AFAIK, not yet. In case I am mistaken, it is better to land this PR for Linux-only because creating conda packages of omniscidb for windows is a non-trivial task on its own.

Reply:
Agree, can we reflect this somehow, eg in PR's name?

Reply:
Sure, done.

Btw, currently, all omniscidb conda packages, except pyomniscidb, are Linux-only.

Reply:
Yeah, I think this is ok for now